### PR TITLE
wap logo and link to msg the person has been added

### PR DESCRIPTION
**Pull request by nittesh31**
whatsapp logo with link has been added , link to the owners whatsapp chat box.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -66,35 +66,45 @@
...
     </div>
-  </footer>
+    <a href="https://api.whatsapp.com/send/?phone=919500074601&text=Hey!+How+can+I+help+you?&app_absent=0"
+      style="position:fixed;bottom:15px;right:15px">
+      <img src="img/wapimg.png" style="width: 50px">
+    </a>
+
+
+
+
```
Comment (chan007) [inline templates/base.html]: remove extra lines

File: templates/base.html [hunk 2]
```diff
@@ -66,35 +66,45 @@
...
             </div>
-            <div class="col-4 footer__copy-text">COPYRIGHT © 2020–2050 QRS Pvt. Ltd</div>
+          </div>
         </div>
+        <div class="col-4 footer__copy-text">COPYRIGHT © 2020–2050 QRS Pvt. Ltd</div>
       </div>
     </div>
-  </footer>
+    <a href="https://api.whatsapp.com/send/?phone=919500074601&text=Hey!+How+can+I+help+you?&app_absent=0"
+      style="position:fixed;bottom:15px;right:15px">
```
Comment (chan007) [inline templates/base.html]: Move this to footer.scss. The style part.

**Review by chan007 — CHANGES_REQUESTED** [review]
Looks good overall

File: templates/base.html (hunk 1 above)
Comment (nittesh31) [reply templates/base.html]: done

File: templates/base.html (hunk 2 above)
Comment (nittesh31) [reply templates/base.html]: done

**Review by chan007 — APPROVED** [review]

**Pull request merged**